### PR TITLE
Modified codecoverage to run netcore tests.

### PR DESCRIPTION
**Pull request by cijothomas**
Only change is adjusting the path of netcore tests.

NetCore.Tests\netcoreapp1.0\

### CODE REVIEW

**Comment by cijothomas** [comment]
This is not going to run the netcore tests, as it require different configs. Aborting this PR.

**Comment by cijothomas** [comment]
I am removing netcore tests from codecoverage tool instead of letting it fail (its failing now).

File: CodeCov.ps1 [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
...or\Net40.Tests\Microsoft.ApplicationInsights.DependencyCollector.Net40.Tests.dll ..\a\Release\Src\DependencyCollector\Net45.Tests\Microsoft.ApplicationInsights.DependencyCollector.Net45.Tests.dll   ..\a\Release\Src\DependencyCollector\Nuget.Tests\Microsoft.ApplicationInsights.DependencyCollector.NuGet.Tests.dll ..\a\Release\Src\PerformanceCollector\Unit.Tests\Unit.Tests.dll ..\a\Release\Src\PerformanceCollector\Xdt.Tests\Xdt.Tests.dll ..\a\Release\Src\Web\Web.Net40.Tests\Microsoft.ApplicationInsights.Web.Net40.Tests.dll ..\a\Release\Src\Web\Web.Net45.Tests\Microsoft.ApplicationInsights.Web.Net45.Tests.dll ..\a\Release\Src\Web\Web.Nuget.Tests\Microsoft.ApplicationInsights.Platform.Web.Nuget.Tests.dll ..\a\Release\Src\WindowsServer\WindowsServer.Net40.Tests\WindowsServer.Net40.Tests.dll ..\a\Release\Src\WindowsServer\WindowsServer.Net45.Tests\WindowsServer.Net45.Tests.dll  ..\a\Release\Src\WindowsServer\WindowsServer.Nuget.Tests\WindowsServer.Nuget.Tests.dll /TestCaseFilter:TestCategory!=Required_4_5_1 /logger:trx" "-filter:+[Microsoft.AI*]* -[*Tests]* -[*TestFramework*]* -[*]Microsoft.ApplicationInsights.Extensibility.Implementation.External*" -hideskipped:All -output:.\coverage.xml
```
Comment (dnduffy) [inline CodeCov.ps1]: Just to be safe for future, can you comment that the netcore tests are run from a separate build step and that is why they are excluded here?

Comment (cijothomas) [inline CodeCov.ps1]: This is not the unit test, this is code coverage step. Which tests are run in build machine - this information is not in code anywhere, but in the build defition itself.

**Pull request merged**